### DISCUSSION TITLE
If a plugin throws, why is the error message prepended with "unknown:"?

### DISCUSSION THREAD

**Discussion by codepunkt** [babel/babel#14855] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/603683/184655482-6719d60e-369c-41ea-aa6d-2ed5d6c8225f.png)
See this image for reference.

**Comment by JLHwung** [comment, marked as answer]
The `"unknown"` should have been the filename of the input source. When the `filename` transform option is not set, Babel will fall back to `"unknown"`.

**Reply by codepunkt** [reply]
Awesome, thanks Huáng!
